SDK_dotnet/GluwaClient.cs

```diff
@@ -0,0 +1,402 @@
+                mBaseUrl = "https://sandbox.api.gluwa.com";
+            }
+            else
+            {
+                mBaseUrl = "https://api.gluwa.com";
+            }
+        }
+
+        public async Task<Result<BalanceResponse>> GetAddressAsync(ECurrency? currency)
+        {
+            var result = new Result<BalanceResponse>();
+            string requestUri = $"{mBaseUrl}/v1/{currency}/Addresses/{mAddress}";
+
+            using (HttpClient httpClient = new HttpClient())
+            {
+                HttpResponseMessage message = await httpClient.GetAsync(requestUri);
+                if (message.IsSuccessStatusCode)
+                {
+                    BalanceResponse balanceResponse = await message.Content.ReadAsAsync<BalanceResponse>();
+                    result.IsSuccess = true;
+                    result.Data = balanceResponse;
+
+                    return result;
+                }
+                else
+                {
+                    throw new HttpListenerException((int)message.StatusCode, message.ReasonPhrase);
+                }
+            }
+        }
+
+        public async Task<Result<List<TransactionResponse>>> GetListTransactionsAsync(
+            ECurrency? currency,
+            uint limit = 100,
+            ETransactionStatusFilter status = ETransactionStatusFilter.Confirmed,
+            uint offset = 0
+            )
+        {
+            var result = new Result<List<TransactionResponse>>();
+
+            string requestUri = $"{mBaseUrl}/v1/{currency}/Addresses/{mAddress}/Transactions";
+
+            var queryParams = new List<string>();
+            if (offset > 0)
+            {
+                queryParams.Add($"offset={offset}");
+            }
+            if (limit > 0)
+            {
+                queryParams.Add($"limit={limit}");
+            }
+            if (queryParams.Any())
+            {
+                queryParams.Add($"status={status}");
+                requestUri = $"{requestUri}?{string.Join("&", queryParams)}";
+            }
+
+            using (HttpClient httpClient = new HttpClient())
+            {
+                httpClient.DefaultRequestHeaders.Add("X-REQUEST-SIGNATURE", getTimestampSignature());
+                HttpResponseMessage message = await httpClient.GetAsync(requestUri);
+                if (message.IsSuccessStatusCode)
+                {
+                    List<TransactionResponse> transactionResponse = await message.Content.ReadAsAsync<List<TransactionResponse>>();
+                    result.IsSuccess = true;
+                    result.Data = transactionResponse;
+
+                    return result;
+                }
+                else
+                {
+                    throw new HttpListenerException((int)message.StatusCode, message.ReasonPhrase);
+                }
+            }
+        }
+
+        public async Task<Result<TransactionResponse>> GetListTransactionDetailAsync(ECurrency? currency, string txnHash)
```

Comment:
When testing, the address format is wrong, but I get a 403 error. What am I doing wrong?

Reply:
Don't know what this means by "address format is wrong." Can you provide more details vis slack?

Reply:
Sorry. I was mistaken. No problem.